src/app/home/home.module.ts

```diff
@@ -1,21 +1,17 @@
 import { IonicModule } from '@ionic/angular';
-import { RouterModule } from '@angular/router';
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { FormsModule } from '@angular/forms';
+import { SharedModule } from '@shared/shared.module';
 import { HomeComponent } from './home.component';
-//import { ActivityCardComponent } from '../components/activity-card/activity-card.component';
+import { HomeRoutingComponent } from './home-routing.component';
+import { HomeRoutingModule } from './home-routing.module';
 import { NgCircleProgressModule } from 'ng-circle-progress';
-import { TodoCardComponent } from '../components/todo-card/todo-card.component';
-import { ActivityCardModule } from '../components/activity-card/activity-card.module';
-
 
 @NgModule({
   imports: [
-    IonicModule,
-    CommonModule,
```

Comment:
Since we've removed them here, we don't need the `import {} ` statement at the top as well